specification.md

```diff
@@ -0,0 +1,1243 @@
+
+Additionally, a Flow Results data package must conform to these additional requirements:
+
+## Components
+
+A Flow Results data package consists of one [Descriptor](https://specs.frictionlessdata.io/data-package/#descriptor), which contains exactly one [Resource](https://specs.frictionlessdata.io/data-resource/) describing the interaction results.  
+
+## Descriptor
+
+The Descriptor JSON object must contain the following required metadata properties:
+
+<table>
+  <tr>
+    <td>Property</td>
+    <td>Description</td>
+    <td>Example</td>
+  </tr>
+  <tr>
+    <td>`profile`</td>
+    <td>Indicates this package is a Flow Results data package. Must be in the format of the example.</td>
+    <td>'flow-results-package'</td>
+  </tr>
+  <tr>
+    <td>`flow_results_specification_version`</td>
+    <td>Indicates the version of this specification the package is compliant with. The Flow Results specification adheres to semantic versioning.</td>
+    <td>'1.0.0-rc1'</td>
+  </tr>
+  <tr>
+    <td>`created`</td>
+    <td>The timestamp for when this package was created/published. This must be in the format of RFC 3339, section 5.6, "date-time".</td>
+    <td>'2017-06-30 15:35:27+00:00'</td>
+  </tr>
+  <tr>
+    <td>`modified`</td>
+    <td>The latest timestamp for when the data in this package was refreshed/updated/changed. This must be in the format of RFC 3339, section 5.6, "date-time".</td>
+    <td>'2017-06-30 15:38:05+00:00'</td>
+  </tr>
+</table>
+
+
+The following metadata properties are recommended, consistent with the Data Packages specification:  (TODO: Should any of these be required?)
+
+<table>
+  <tr>
+    <td>Property</td>
+    <td>Description</td>
+    <td>Example</td>
+  </tr>
+  <tr>
+    <td>`name`</td>
+    <td>A short url-usable (and preferably human-readable) name of the package. This MUST be lower-case and contain only alphanumeric characters along with ".", "_" or "-" characters. It will function as a unique identifier and therefore SHOULD be unique in relation to any registry in which this package will be deposited (and preferably globally unique).</td>
+    <td>"flow-results-demo-package"</td>
+  </tr>
+  <tr>
+    <td>`id`</td>
+    <td>A property reserved for globally unique identifiers. Examples of identifiers that are unique include UUIDs and DOIs.
+
+A common usage pattern for Data Packages is as a packaging format within the bounds of a system or platform. In these cases, a unique identifier for a package is desired for common data handling workflows, such as updating an existing package. While at the level of the specification, global uniqueness cannot be validated, consumers using the id property MUST ensure identifiers are globally unique.</td>
+    <td>"b03ec84-77fd-4270-813b-0c698943f7ce"</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+    <td></td>
+  </tr>
+</table>
+
+
```

Comment:
What about including other optional fields to better describe the dataset? Things like period of time of the capture, location, type of channels used, etc? Also, do we want to allow custom fields? If that's the case, I'd suggest mandating them be prefixed with 'X-' for example to avoid clashes in the future. I can open a separate issue on this there is enough interest.